From 88c452b5df8eb971c8022cdab9d0c22a884d30e3 Mon Sep 17 00:00:00 2001
From: Thomas Goirand <zigo@debian.org>
Date: Thu, 8 Oct 2015 11:06:35 -0700
Subject: Removes deprecation warning

 The depraction warning is flooding Nova logs. As it's too late to fix Nova,
 best way forward is to just remove the depraction warning, which this patch
 does.
Bug-Debian: https://bugs.debian.org/774966
Forwarded: not-needed
Last-Update: 2014-01-17

Patch-Name: remove-deprecation-warning.patch
---
 amqp/channel.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/amqp/channel.py b/amqp/channel.py
index 05eb09a..69ee6c1 100644
--- a/amqp/channel.py
+++ b/amqp/channel.py
@@ -612,9 +612,6 @@ class Channel(AbstractChannel):
         args.write_table(arguments)
         self._send_method((40, 10), args)
 
-        if auto_delete:
-            warn(VDeprecationWarning(EXCHANGE_AUTODELETE_DEPRECATED))
-
         if not nowait:
             return self.wait(allowed_methods=[
                 (40, 11),  # Channel.exchange_declare_ok
